README.md

```diff
@@ -147,7 +147,7 @@ Proxy options:
 
     --direct-domains [-]<regexp>,... (env FORWARDER_DIRECT_DOMAINS)
         Connect directly to the specified domains without using the upstream proxy. Prefix domains with '-' to exclude
-        requests to certain domains from being directed.This flag takes precedence over the PAC script.
+        requests to certain domains from being directed. This flag takes precedence over the PAC script.
```

Comment:
Good catch. It looks like it comes from invalid usage string in `bind/flag.go`. Could you address it there as well?

Reply:
Oh that's what you did.